packages/notebook-extension/schema/tracker.json

```
@@ -746,9 +746,9 @@
         "showProgress": true
       }
     },
-    "experimentalDisableDocumentWideUndoRedo": {
-      "title": "Experimental settings to enable the undo/redo on the notebook document level.",
-      "description": "Disable the undo/redo on the notebook document level, so actions independent cells can have their own history. The undo/redo never applies on the outputs, in other words, outputs don't have history. A moved cell completely looses history capability for now.",
+    "experimentalEnableDocumentWideUndoRedo": {
+      "title": "Enable the undo/redo on the notebook document level.",
+      "description": "Experimental settings that enables the undo/redo at the notebook document level; aka undoing within a cell may undo the latest notebook change that happen in another cell.",
```

Should we drop experimental? I guess this could be done in a separate PR.

```suggestion
      "description": "Experimental settings that enables the undo/redo at the notebook document level; aka undoing within a cell may undo the latest notebook change that happened in another cell.",
```

---

The undo/redo at document level does not feel user friendly. So I kept the _experimental_. But I guess we could remove it and iterate when we get feedback.

---

If we are considering dropping it we could add "deprecated, will be removed in 5.0" in the description and see if any one complains. If no one does we can drop it.